go/vt/srvtopo/resilient_server.go

```diff
@@ -128,13 +137,25 @@ type srvKeyspaceNamesEntry struct {
 	// the mutex protects any access to this structure (read or write)
 	mutex sync.Mutex
 
+	// refreshingChan is used to synchronize requests and avoid hammering
+	// the topo server
+	refreshingChan chan bool
+
 	insertionTime time.Time
 	lastQueryTime time.Time
 	value         []string
 	lastError     error
 	lastErrorCtx  context.Context
 }
 
+const (
+	watchStateIdle = iota
+	watchStateStarting
+	watchStateRunning
+)
+
+type watchState int
```

Comment:
nit: you can declare that one before the const, and then type the consts:

const (
    watchStateIdle watchState = iota
    watchStateStarting
    watchStateRunning
)

Reply:
Done.